src/main.kt

```diff
@@ -14,6 +13,10 @@ class DokkaArguments {
     ValueDescription("<path>")
     public var src: String = ""
 
+    Argument(value = "srcLink", description = "Mapping between a source directory and a Web site for browsing the code")
+    ValueDescription("<path>=<url>[#lineSuffix]")
+    public var srcLink: String = ""
+
```

Comment:
Can't there be multiple mappings? E.g. when project consists of multiple source repositories checked out into single folder structure.